src/styles/blocks/card.scss

```
@@ -0,0 +1,115 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+h2 {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+
+.product-card {
+  width: 200px;
+
+
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background: var(--white-accent);
+}
+
+
+
```

```suggestion
.product-card {
  width: 200px;

  border-radius: 5px;
  border: 1px solid #F3F3F3;
  background: var(--white-accent);
}

```
no more than 1 blank line